src/umpire/resource/CMakeLists.txt

```diff
@@ -7,21 +7,27 @@
 set (umpire_resource_headers
   DefaultMemoryResource.hpp
   DefaultMemoryResource.inl
+  FileMemoryResource.hpp
   HostResourceFactory.hpp
   MemoryResource.hpp
   MemoryResourceFactory.hpp
   MemoryResourceRegistry.hpp
   MemoryResourceTypes.hpp
   NullMemoryResource.hpp
   NullMemoryResourceFactory.hpp
+  FileMemoryResource.hpp
```

Comment:
You've added this file twice. Can you remove the duplicated, and make sure they are sorted alphabetically.